src/components/crossDevice/MobileComplete/index.js

```diff
@@ -1,12 +1,62 @@
-import { h } from 'preact'
+import { h, Component } from 'preact'
 
+import VerificationComplete from '../../Complete/VerificationComplete.js'
 import { trackComponent } from '../../../Tracker'
+import {preventDefaultOnClick} from '../../utils'
+import theme from '../../Theme/style.css'
+import style from './style.css'
 
+class MobileComplete extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {verificationComplete: false}
+  }
 
-const MobileComplete = () => {
-  return (
-    <div>Complete!</div>
-  )
+  shouldDisplayCompleteNotice = () => {
```

Comment:
I would do this at history level, to keep both `MobileComplete` and `VerificationComplete` dumber, in order to reduce inter dependency.